docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,275 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_px4_flight_controller:
+
+****************************************************
+Setting up PX4 Firmware for Use with IFCI
+****************************************************
+.. Converted from Jordan's google docs tutorial
+
+This tutorial will cover how to build and setup PX4 with Vertiq IQUART integrations including the :ref:`IQUART Flight Controller Interface (IFCI)<controlling_ifci>`. These integrations allow multiple Vertiq 23-06 and 23-14 modules to be connected to a single PX4 UART and have the flight controller control them, change their settings, and receive telemetry.
+
+.. note::
+    Our IQUART integrations are not currently included in mainline PX4, but there is a pull request being reviewed
+.. set up motor -- all done in ifci px4 flight controller or 
+..      ensure the motor is flashed with underactuated
+..      If using multiple motors ensure that the IDs are different
+..      Set the baudrate to 921600 or higher
+..      Ensure the proper default file is selected
+..      point forward and zero offset
+..      More information can be found here *up12 setup page*
+..  
+.. set up px4 -- all done in ifci px4 flight controller
+..      assume flashed with correct stuff
+..      enable Vertiq stuff on correct port
+..          vertiq io cfg? 
+..      set vertiq_baud to matching baud (921600)
+..      set num cvs
+..      set telemetry ids
+..      set Disarm trigger
+
+Setting Up the PX4 Toolchain
+=============================
+
+.. This can section can be removed once we are integrated in
+
+Before beginning, the PX4 toolchain must be installed. We recommend that you follow the `PX4 guides <https://docs.px4.io/main/en/dev_setup/dev_env.html>`_ for this, but replace the step where the repository is cloned using ``git clone https://github.com/PX4/PX4-Autopilot.git --recursive`` with ``git clone https://github.com/iq-motion-control/PX4-Autopilot --recursive``.
```

Comment:
I don't think enough attention is called to the swapping stuff out for the fork part here, I actually missed it on my first read through. May want to make it more obvious that people need to swap out the address, they cannot just follow the PX4 guide. In my experience, users do not faithfully read every sentence sequentially on these pages, so a big bright note box or some bold text calling attention to this may help avoid some people glancing pas this.

Reply:
Just added this with the most recent commit. Good call out